### PR TITLE
Add logging metrics

### PR DESCRIPTION
**Pull request by pavelzubarev**
Sometimes it's handy to monitor logs severity produced. Adding new optional config parameter

### CODE REVIEW

File: cognite/extractorutils/configtools.py [hunk 1]
```diff
@@ -191,6 +192,7 @@ class LoggingConfig:
 
     console: Optional[_ConsoleLoggingConfig]
     file: Optional[_FileLoggingConfig]
+    log_metrics: Optional[bool] = False  # enables metrics on the number of log messages recorded (per logger and level)
```
Comment (pavelzubarev) [inline cognite/extractorutils/configtools.py:195]: I'm not sure it's the best name for the config parameter if you have better ideas, shoot!

File: cognite/extractorutils/logging_prometheus.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import logging
+
+from prometheus_client import Counter
+
+log_entries = Counter(
+    "cognite_python_logging_messages_total", "Count of log entries by logger and level.", ["logger", "level"]
```
Comment (chrfla-cognite) [inline cognite/extractorutils/logging_prometheus.py:6]: @pavelzubarev should we change the name to `cognite_python_logging_messages`, so don't end up with `_total_total` in the name.

**Review by lprovensi — APPROVED** [review]
looks good 👍

File: cognite/extractorutils/logging_prometheus.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class ExportingLogHandler(logging.Handler):
+    """A LogHandler that exports logging metrics for Prometheus.io."""
+
+    def emit(self, record):
+        log_entries.labels(record.name, record.levelname).inc()
+
+
+def export_log_stats_on_root_logger(logger=logging.getLogger()):
```
Comment (chrfla-cognite) [inline cognite/extractorutils/logging_prometheus.py:17]: @pavelzubarev will this get the correct logger?

File: cognite/extractorutils/configtools.py (hunk 1 above)
Comment (chrfla-cognite) [reply cognite/extractorutils/configtools.py:195]: It makes sense, after reading the comment

File: cognite/extractorutils/logging_prometheus.py (hunk 3 above)
Comment (pavelzubarev) [reply cognite/extractorutils/logging_prometheus.py:17]: yes. that will get the root logger by default

File: cognite/extractorutils/logging_prometheus.py (hunk 2 above)
Comment (pavelzubarev) [reply cognite/extractorutils/logging_prometheus.py:6]: good catch!

**Review by chrfla-cognite — APPROVED** [review]

**Review by lprovensi — APPROVED** [review]

**Pull request merged**